docs/src/app/components/pages/components/DropDownMenu/README.md

```diff
@@ -0,0 +1,6 @@
+## Drop Down Menu
+
+To find out more about the `DropDownMenu` component please visit the Material Design's
+specifications **WHERE IS IT IN THE SPEC?!**.
```

Comment:
You are right, https://github.com/callemall/material-ui/issues/2416 makes a lot of sense :+1:.
Let's wait 0.14 before taking care of it.